src/pycram/designators/action_designator.py

```diff
@@ -978,78 +984,87 @@ def perform(self) -> None:
             grasp orientation, computing slice positions, and orchestrating the necessary robot motions
             for executing the cutting task.
             """
-
+            print(self.technique)
+            # Store the object and tool at the time of execution
             self.object_to_be_cut_at_execution = self.object_to_be_cut.data_copy()
             self.tool_at_execution = self.tool.data_copy()
 
-            # Get grasp orientation and target pose
+            # Obtain the preferred grasp orientation for the cutting action
             grasp = robot_description.grasps.get_orientation_for_grasp("top")
-            # Retrieve object and robot from designators
+
+            # Access the designated object for cutting and retrieve its dimensions
             object = self.object_to_be_cut.bullet_world_object
             obj_dim = object.get_object_dimensions()
 
+            # Arrange object dimensions in order of size for cutting calculations
             dim = [max(obj_dim[0], obj_dim[1]), min(obj_dim[0], obj_dim[1]), obj_dim[2]]
+
+            # Retrieve the current pose of the object and transform it to the object frame
             oTm = object.get_pose()
             object_pose = object.local_transformer.transform_to_object_frame(oTm, object)
 
-            # from bread_dim calculate def a calculation that gets me the highest number from the first 2 entries
-            # Given slice thickness is 3 cm or 0.03 meters
-            slice_thickness = self.slice_thickness
-            # Calculate slices and transform them to the map frame with orientation
-            obj_length = dim[0]
-            obj_width = dim[1]
-            obj_height = dim[2]
+            # Set the slice thickness for the cutting operation, defaulting to 3 cm if not specified
+            slice_thickness = self.slice_thickness or 0.03  # Default slice thickness in meters
```

Comment:
Why is the slice thickness set in the execution here? Wouldnt it be cleaner to default it in line 963?